tb_plugin/torch_tb_profiler/profiler/module_parser.py

```diff
@@ -181,12 +181,18 @@ def parse_kernels(kernel_list):
             for kernel in kernel_list:
                 dur = kernel.end_time - kernel.start_time
                 op_name = "N/A" if kernel.op_node is None else kernel.op_node.name
-                key = kernel.name + "###" + op_name
+                key = kernel.name + "###" + op_name + "###" + \
```

Comment:
use string.join instead of add operation?

Reply:
Thanks. Done.